client/styles/main.css

```diff
@@ -27,6 +27,19 @@ main {
     font-family: Lato;
     src: url('../assets/fonts/lato/Lato-Regular.ttf');
 }
+
+@font-face {
+    font-family: Lato;
+    src: url('../assets/fonts/lato/Lato-Bold.ttf');
+    font-weight: bold;
```

Comment:
we have multiple weight. the default is the regular. but if we want to use bold, is load to.